activemodel/lib/active_model.rb

```diff
@@ -47,6 +47,7 @@ module ActiveModel
   autoload :SecurePassword
   autoload :Serialization
   autoload :Translation
+  autoload :Type
```

Comment:
I think it should be inside `eager_autoload`

Reply:
`ActiveModel::Type` is not always used when using ActiveModel.
Should we eager_load even though?

Reply:
Yeah. I don't think so.